### PR TITLE
add missing wge releases

### PR DESCRIPTION
**Pull request by ahsayde**
Closes

**What changed?**
add releases `v0.10.1` and  `v0.10.2` to docs

**Why was this change made?**

**How was this change implemented?**

**How did you validate the change?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: website/docs/enterprise/releases.mdx [hunk 1]
```diff
@@ -7,6 +7,50 @@ import TierLabel from "../_components/TierLabel";
...
 # Releases <TierLabel tiers="enterprise" />
 
+## v0.10.2
+2022-11-15
+
+### Highlights
+- Retain template parameter ordering.
+- Allow configuration of the delimiters in templates.
+- Add create a pipeline button.
+- add missing support for policy version v2beta2 to tenancy cmd.
```
Comment (serboctor) [inline website/docs/enterprise/releases.mdx:17]:
```suggestion
- Add missing support for policy version v2beta2 to tenancy cmd.
```

File: website/docs/enterprise/releases.mdx [hunk 2]
```diff
@@ -7,6 +7,50 @@ import TierLabel from "../_components/TierLabel";
...
+- cluster-bootstrap-controller v0.3.0
+- (optional) policy-agent 2.1.1
+
+## v0.10.1
+2022-11-10
+
+### Highlights
+
+- Create non-cluster resources / Add Edit option to resources with create-request annotation
+- bump pipeline-controller
```
Comment (serboctor) [inline website/docs/enterprise/releases.mdx:31]:
```suggestion
- Bump pipeline-controller
```

File: website/docs/enterprise/releases.mdx [hunk 3]
```diff
@@ -7,6 +7,50 @@ import TierLabel from "../_components/TierLabel";
...
+- Parse annotations from template
+- Add cost estimate message if available
+- Adds support for showing policy modes and policy configs in the UI
```
Comment (serboctor) [inline website/docs/enterprise/releases.mdx:34]:
```suggestion
- Add support for showing policy modes and policy configs in the UI 
```

File: website/docs/enterprise/releases.mdx [hunk 4]
```diff
@@ -7,6 +7,50 @@ import TierLabel from "../_components/TierLabel";
...
+- bump pipeline-controller
+- Parse annotations from template
+- Add cost estimate message if available
+- Adds support for showing policy modes and policy configs in the UI 
+
+- Show suspended status on pipelines detail
+- YAML view for Pipelines
+- Align and link logo
+
+- Actually remove the watcher from the helm-watcher-cache
```
Comment (serboctor) [inline website/docs/enterprise/releases.mdx:40]:
```suggestion
- Remove the watcher from the helm-watcher-cache
```

File: website/docs/enterprise/releases.mdx [hunk 5]
```diff
@@ -7,6 +7,50 @@ import TierLabel from "../_components/TierLabel";
...
+- UI 1817 disable create target name space if name space is flux system
+
+- Adding edit capi cluster resource acceptance test
```
Comment (serboctor) [inline website/docs/enterprise/releases.mdx:43]:
```suggestion
- Add edit capi cluster resource acceptance test
```

**Review by serboctor — CHANGES_REQUESTED** [review]
Minor capatalizations and changes

**Review by foot — APPROVED** [review]
🌟 Fantastic, thanks gang!

**Review by serboctor — APPROVED** [review]

**Pull request merged**